data/sql/updates/pending_db_world/rev_1495464409315344160.sql

```diff
@@ -0,0 +1,147 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1495464409315344160');
+
+-- The Troll Cave requires 14 kills, not 10.
+UPDATE `quest_template` SET `RequiredNpcOrGoCount1` = 14 where `ID` = 182;
+
+-- Make The Grizzled Den more tolerable.
+DELETE FROM `creature` WHERE `guid` IN (3137,3255,3261,3262,3548,3551,3554,3558,3583,3239,3243,3246,3257,3258,3552,3565,3566,3567,3570,3573,3574,3578);
+
+-- The Perfect Stout does not require Rejold's New Brew, it's the otherway around.
+UPDATE `quest_template_addon` SET `NextQuestID` = 0, `PrevQuestID` = 315 WHERE `ID` = 415;
+
+-- All Tunnel Rats should be able to drop ears, not just 1 out of 6 types.
+INSERT INTO `creature_loot_template` VALUES
+(1172, 3110,-38,1,0,1,1),
+(1174, 3110,-80,1,0,1,1),
+(1175, 3110,-80,1,0,1,1),
+(1176, 3110,-38,1,0,1,1),
+(1177, 3110,-80,1,0,1,1)
+
+-- Red Silk Bandanas needs The Defias Brotherhood (6), not Red Leather Bandanas.
+UPDATE `quest_template_addon` SET `PrevQuestID` = 155 WHERE `ID` = 214;
+
+-- Letter to Stormpike follows Encrypted Letter
+UPDATE `quest_template_addon` SET `PrevQuestID` = 511 WHERE `ID` = 514;
+
+-- Kurzen Jungle Fighter also drops Jungle Remedy
+INSERT INTO `creature_loot_template` VALUES (937,2633,-33,1,0,1,1);
+
+-- Inspecting the ruins requires either They Call Him Smilling Jim or James Hyal
+UPDATE `quest_template_addon` SET `NextQuestID` = 11123 WHERE `ID` = 1282;
+UPDATE `quest_template_addon` SET `NextQuestID` = 11123 WHERE `ID` = 1302;
+UPDATE `quest_template_addon` SET `PrevQuestID` =     0 WHERE `ID` = 11123;
+
+-- Raising the drop rate on Forked Mudrock Tongues
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = -40 WHERE `entry` IN 4397 AND `item` = 5883;
```

Comment:
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='-40' WHERE (`entry`='4397') AND (`item`='5883')